index.html

```diff
@@ -1,1227 +1,1567 @@
+					<source srcset="img/questions/bg/x480/cloudsBottom_x480.png" type="image/jpeg"
+						media="(min-width:480px)">
+					<img class="question__bg_cloudsBottom" src="img/questions/bg/x320/cloudsBottom_x320.png"
+						alt="clouds bottom">
+				</picture>
+			</div>
+			<div class="questions__container">
+				<picture>
+					<source srcset="img/questions/bg_elements/x1920/girlWithHorse_x1920.png" type="image/jpeg"
+						media="(min-width:1920px)">
+					<source srcset="img/questions/bg_elements/x1000/girlWithHorse_x1000.png" type="image/jpeg"
+						media="(min-width:1000px)">
+					<source srcset="img/questions/bg_elements/x768/girlWithHorse_x768.png" type="image/jpeg"
+						media="(min-width:768px)">
+					<source srcset="img/questions/bg_elements/x480/girlWithHorse_x480.png" type="image/jpeg"
+						media="(min-width:480px)">
+					<img class="questions__girl_with_horse" src="img/questions/bg_elements/x320/girlWithHorse_x320.png"
+						alt="girlWithHorse">
+				</picture>
+				<picture>
+					<source srcset="img/questions/bg_elements/x1920/bush_x1920.png" type="image/jpeg"
+						media="(min-width:1920px)">
+					<source srcset="img/questions/bg_elements/x1000/bush_x1000.png" type="image/jpeg"
+						media="(min-width:1000px)">
+					<img class="questions__bush" src="img/questions/bg_elements/x1920/bush_x1920.png" alt="bush">
+				</picture>
+				<picture>
+					<source srcset="img/questions/bg_elements/x1920/frame_x1920.png" type="image/jpeg"
+						media="(min-width:1920px)">
+					<source srcset="img/questions/bg_elements/x1000/frame_x1000.png" type="image/jpeg"
+						media="(min-width:1000px)">
+					<source srcset="img/questions/bg_elements/x768/frame_x768.png" type="image/jpeg"
+						media="(min-width:768px)">
+					<source srcset="img/questions/bg_elements/x480/frame_x480.png" type="image/jpeg"
+						media="(min-width:480px)">
+					<img class="questions__frame" src="img/questions/bg_elements/x320/frame_x320.png" alt="frame">
+				</picture>
+				<div class="questions__form_container">
+					<form class="questions__form">
 						<p class="form__header">Отвечаем на вопросы</p>
-						<p class="form__description">Мы ответим на все ваши вопросы по телефону, или вы можете записаться на <b>бесплатное</b> пробное занятие</p>
-						<input name="user_name" class="form__input" id="userName" type="text" placeholder="Ваше имя" pattern="^[а-яА-ЯёЁa-zA-ZЁёЇїІіЄєҐґ']+$" invalidTooltip = "Вводите буквы" required>
-						<input name="user_phone" class="form__input" id="userPhone" type="tel" placeholder="Ваш телефон" pattern="^[+\-\s()0-9]{1,20}" invalidTooltip = "Вводите цифры" required>
+						<p class="form__description">Мы ответим на все ваши вопросы по телефону, или вы можете
+							записаться на <b>бесплатное</b> пробное занятие</p>
+						<div class="input_container">
+							<p class="valid_tooltip write_letters">Вводите буквы</p>
+							<input class="form__input" type="text" placeholder="Ваше имя" data-name data-to-Validate>
```

Comment:
you don't need to use capital letters in data attribute, anyway you will give "toValidate" in js

Reply:
Done.